graphql_api/types/owner/owner.graphql

```diff
@@ -39,4 +39,5 @@ type Owner {
   yaml: String
   aiFeaturesEnabled: Boolean!
   aiEnabledRepos: [String]
+  uploadTokenRequired: Boolean
```

Comment:
Is it intentional for this to be null?

Reply:
Intertactor should always return boolean, but the decorator `@require_part_of_org` should return null if user is not part of the org!